src/Toolkit/A.php

```diff
@@ -351,6 +351,31 @@ public static function has(
 		return in_array($value, $array, $strict);
 	}
 
+	/**
+	 * Join array elements as a string,
+	 * also supporting nested arrays
+	 */
+	public static function implode(
+		array $array,
+		string $separator = ', '
```

Comment:
For consistency with the core PHP `implode()` function:

```suggestion
		string $separator = ''
```